aptos-move/framework/aptos-stdlib/sources/cryptography/bls12381.spec.move

```diff
@@ -1,46 +1,95 @@
 spec aptos_std::bls12381 {
     spec aggregate_pubkeys_internal {
-        // TODO: temporary mockup.
         pragma opaque;
+        ensures result_1 == spec_aggregate_pubkeys_internal_1(public_keys);
+        ensures result_2 == spec_aggregate_pubkeys_internal_2(public_keys);
     }
 
     spec aggregate_signatures_internal {
-        // TODO: temporary mockup.
         pragma opaque;
+        ensures result_1 == spec_aggregate_signatures_internal_1(signatures);
+        ensures result_2 == spec_aggregate_signatures_internal_2(signatures);
     }
 
     spec validate_pubkey_internal {
-        // TODO: temporary mockup.
         pragma opaque;
+        ensures result == spec_validate_pubkey_internal(public_key);
     }
 
     spec signature_subgroup_check_internal {
-        // TODO: temporary mockup.
         pragma opaque;
+        ensures result == spec_signature_subgroup_check_internal(signature);
     }
 
     spec verify_aggregate_signature_internal {
-        // TODO: temporary mockup.
         pragma opaque;
+        ensures result == spec_verify_aggregate_signature_internal(aggsig, public_keys, messages);
     }
 
     spec verify_multisignature_internal {
-        // TODO: temporary mockup.
         pragma opaque;
+        ensures result == spec_verify_multisignature_internal(multisignature, agg_public_key, message);
     }
 
     spec verify_normal_signature_internal {
-        // TODO: temporary mockup.
         pragma opaque;
+        ensures result == spec_verify_normal_signature_internal(signature, public_key, message);
     }
 
     spec verify_proof_of_possession_internal {
-        // TODO: temporary mockup.
         pragma opaque;
+        ensures result == spec_verify_proof_of_possession_internal(public_key, proof_of_possesion);
     }
 
     spec verify_signature_share_internal {
-        // TODO: temporary mockup.
         pragma opaque;
+        ensures result == spec_verify_signature_share_internal(signature_share, public_key, message);
     }
+
+    // ----------------
+    // Helper functions
+    // ----------------
+
```

Comment:
These are all uninterpreted functions, which I think is correct, but this should be documented. You are also not using the expected markdown I believe -- can't you use ###... somehow for sectioning?

Reply:
@rahxephon89 , here is an example of sectioning: https://github.com/aptos-labs/aptos-core/blob/main/third_party/move/documentation/examples/diem-framework/move-packages/DPN/sources/Diem.move#L1603

Reply:
thanks @junkil-park! the comment has been fixed